### PR TITLE
{chem}[intel/2019a] OpenBabel v2.4.1 w/ Python 3.7.2

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/o/OpenBabel/OpenBabel-2.4.1-intel-2019a-Python-3.7.2.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://openbabel.org'
+description = """Open Babel is a chemical toolbox designed to speak the many
+ languages of chemical data. It's an open, collaborative project allowing anyone
+ to search, convert, analyze, or store data from molecular modeling, chemistry,
+ solid-state materials, biochemistry, or related areas."""
+
+toolchain = {'name': 'intel', 'version': '2019a'}
+toolchainopts = {}
```
Comment (akesandgren) [inline easybuild/easyconfigs/o/OpenBabel/OpenBabel-2.4.1-intel-2019a-Python-3.7.2.eb]: Drop empty toolchainopts

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/6fd2ef2cd1e5f38761c3ec63a5830f73 for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/3069ab5da970951c41a592a025aba12c for a full test report.

**Comment by migueldiascosta** [comment]
Going in, thanks @boegel!

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node3152.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/9a142bee9db6cb4511cd346216e5cb53 for a full test report.

**Comment by migueldiascosta** [comment]
hm, 1 test fails on skylake?

**Comment by boegel** [comment]
Yes, indeed...
Not sure how much effort it would be to get that fixed (I won't have time to deep dive into that unfortunately...)

**Comment by migueldiascosta** [comment]
and it actually doesn't fail me on skylake... (test report at https://gist.github.com/migueldiascosta/083d4c69b6fa12a1e282d4fea51152a5)

**Pull request merged**